### PR TITLE
Final Built

### PR DESCRIPTION
**Pull request by muneebulrehman**
Completed building Survey form. This is the final built.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+						<label>
+							<input
+								type="radio"
+								id="difinitely"
+								name="first"
+								value="first"
+								checked
+							/>
+							Definitely</label
+						>
```
Comment (muneebulrehman) [inline index.html:96]: Please use proper indentation. You can use prettier extension in vs code which would automatically format your code with nice and beautiful indentation.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background-image:
+    linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(/pexels-max-vakhtbovych-6636262.jpg);
```
Comment (muneebulrehman) [inline style.css:35]: Please use the relative path for image addresses. Add a ' . ' before the ' / '

File: index.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+		<div class="main-div">
```
Comment (muneebulrehman) [inline index.html:27]: Please use semantic html tag

instead of div for independent containers containing multiple divs

File: index.html [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+						</label>
+						<label>
+							<input
+								type="checkbox"
+								id="challenges"
+								name="first"
+								value="first"
+							/>
+							Challenges</label
+						>
```
Comment (muneebulrehman) [inline index.html:143]: Please use proper indentation. You can use prettier extension in vs code which would automatically format your code with nice and beautiful indentation.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+  z-index: -1;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (muneebulrehman) [inline style.css:39]: You can also combine these properties directly in background selector. I suggest you checkout this link for more: https://www.w3schools.com/css/css_background_shorthand.asp

**Review by muneebulrehman — COMMENTED** [review]
# Changes required.

Please check my comments below and make all the required changes before proceeding further.

Submit for re-review when you are done.
Thanks. Happy coding 😊✌
CatComputerGIF
️c

**Review by Rustamxon7 — APPROVED** [review]
## Hi @muneebulrehman   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**